### ISSUE TITLE
`set-state` and `set-output` workflow commands have been deprecated

### ISSUE THREAD

**Issue by localheinz** [shivammathur/setup-php#654]
**Describe the bug**

**Version**
- [x] I have checked releases, and the bug exists in the latest patch version of `v1` or `v2`.
- [ ] `v2`
- [x] `v1`

**Runners**

- [x] GitHub Hosted
- [ ] Self Hosted

**Operating systems**

- `ubuntu-latest`

**PHP versions**

n/a

**To Reproduce**

See https://github.com/sebastianbergmann/phpunit/blob/9.5.25/.github/workflows/ci.yml:

```yaml
jobs:
  coding-guidelines:
    name: Coding Guidelines

    runs-on: ubuntu-latest

    steps:
      - name: Checkout
        uses: actions/checkout@v3

      - name: Install PHP
        uses: shivammathur/setup-php@v2
        with:
          php-version: 8.1
          extensions: none, iconv, json, phar, tokenizer
          coverage: none
          tools: none

      - name: Run PHP-CS-Fixer
        run: ./tools/php-cs-fixer fix --dry-run --show-progress=dots --using-cache=no --verbose
```

**Expected behavior**

No deprecations

**Screenshots/Logs**

See https://github.com/sebastianbergmann/phpunit/actions/runs/3233102102/jobs/5294525446#step:3:14:

![CleanShot 2022-10-12 at 10 31 25@2x](https://user-images.githubusercontent.com/605483/195292500-ef87eaf0-c959-47d0-b0e7-8d2f44a39809.png)

**Additional context**

See https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/

**Are you willing to submit a PR?**

Sure!

**Comment by shivammathur** [comment]
Fixed in https://github.com/shivammathur/setup-php/commit/0575a32108978260a650e390d7008ce2c95b43e3

I will create a new release this weekend.

**Comment by localheinz** [comment]
Thank you, @shivammathur!

**Comment by jrfnl** [comment]
Came here to report the same, only to see it already fixed. Thanks @shivammathur !

**Comment by ramondelafuente** [comment]
Wil this be a v3 release or an update of v2?

**Comment by shivammathur** [comment]
This will be an update of v2.

**Comment by shivammathur** [comment]
Released [2.22.0](https://github.com/shivammathur/setup-php/releases/tag/2.22.0) with the fix.
